release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -199,6 +199,13 @@ For more information, see xref:../networking/hardware_networks/about-sriov.adoc#
 [id="ocp-4-12-nodes"]
 === Nodes
 
+[id="ocp-4-12-node-cron-job-time-zone"]
+==== Cron job time zones (Technology Preview)
+
+Setting a time zone for a cron job's schedule is now offered as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. If a time zone is not specified, the Kubernetes controller manager interprets the schedule relative to its local time zone.
```

Comment:
You could consider changing "a cron job's schedule" to something like one of the following suggestions, to avoid using a possessive:

> the schedule of a cron job
> a cron job schedule

Reply:
Thanks, updating!